README.md

```diff
@@ -1,20 +1,45 @@
-# SNIRF File Format
+# Shared Near Infrared Spectroscopy Format (SNIRF)
 
-## How to download the specification, sample files and parsers
+This repository stores the Shared Near Infrared Spectroscopy Format (SNIRF).  
+SNIRF is designed by the community in an effort to facilitate sharing and analysis of NIRS data.
+
+View the most recent release version of the format: [here](https://github.com/fNIRS/snirf/blob/da550abf7ec70084e31ba428df09a9dcbf3e6036/snirf_specification.md).  
```

Comment:
by release version, do you mean HEAD? or one of these branches?

https://github.com/fNIRS/snirf/branches

Reply:
I mean the last version that was released. So this would be 1.0 soon. Then when 1.1 is released it would be that.

I've currently linked to the Nov 2019 commit based on this comment. https://github.com/fNIRS/snirf/issues/51#issuecomment-876285597

I've added this because most people who are using the spec will not want to know what's in a development branch, as it may change. They want to know about the stable version that they must abide by.

Reply:
Is there a better terminology I should use? I want to make sure this isn't confusing to newcomers. And if you're confused by what I mean, then others will be to